src/unused.js

```diff
@@ -0,0 +1 @@
+console.log('Ooops!);
```

Comment:
It seems like there is a missing closing quote for the string passed to `console.log`. Make sure to close the string with a matching quote to avoid syntax errors.